src/js/main.js

```diff
@@ -19,7 +19,7 @@ Site = {
     $(document).ready(function () {
       _this.SplashVideo.init();
       _this.WhatIsVideo.init();
-      _this.Modules.init();
+      //_this.Modules.init();
```

Comment:
why?

Reply:
Cuz it's broken in my local due to a data structure change